API.md

```diff
@@ -1,19 +1,349 @@
-# Party Service API
+###Party Service API
 
-### The implemented API for this serivce can be found at /swagger/party/party-api.yaml
+######*Acceptance tests can be run to populate a local version with test data.*
```

Comment:
6 hashes... do we really have 6 levels of nesting under the main heading, or is this more _aesthetic_?

In markdown, we care more about hierarchy than the way that the markdown document looks when rendered.